libs/src/api/track.js

```diff
@@ -51,7 +51,7 @@ class Tracks extends Base {
    */
   async getTrendingTracks (time = null, idsArray = null, limit = null, offset = null) {
     this.REQUIRES(Services.IDENTITY_SERVICE)
-    return this.identityService.getTrendingTracks(time, idsArray, limit, offset)
```

Comment:
should remove the identityservice getTrendingTracks probably?

Reply:
The dapp actually needs it for listen counts still.
https://github.com/AudiusProject/audius-dapp/pull/689/files#diff-0cd04d3f79e27382216efd14d61b88e2R480

Reply:
ah nvm